frontend/app/model/MembersOnlyContent.scala

```diff
@@ -0,0 +1,31 @@
+package model
+
+import com.gu.contentapi.client.model.{Asset, Content}
+
+case class MembersOnlyContent(content: Content) {
```

Comment:
No it's okay to leave it but the extractor below isn't necessary. I think we just need to create an apply method instead - we can pair on this if you like?